### DISCUSSION TITLE
Cloning the entire repository for docker compose deployment

### DISCUSSION THREAD

**Discussion by buongiorgio** [ghostfolio/ghostfolio#4096] [Q&A] [answered]
Hi there,
I'm following these instruction https://github.com/ghostfolio/ghostfolio#run-with-docker-compose to install Ghostfolio with docker compose. I'm confused about the bullet point: "Create a local copy of this Git repository (clone)". Is there a reason to clone the whole repository, instead of downloading just the files docker-compose.yml and .env?

**Comment by dtslvr** [comment, marked as answer]
Hi @buongiorgio

You don’t need to clone the whole repository. Downloading the `docker-compose.yml` and `.env` files is enough. Just make sure to adjust any file paths accordingly.
